### PR TITLE
Fix the PolicyGenerator ArgoCD Policy

### PR DESCRIPTION
**Pull request by dhaiducek**
ref: https://issues.redhat.com/browse/ACM-10650

### CODE REVIEW

File: community/CM-Configuration-Management/policy-openshift-gitops-policygenerator.yaml [hunk 1]
```diff
@@ -36,15 +36,14 @@ spec:
                     initContainers:
                     - args:
                       - -c
-                      - cp /etc/kustomize/plugin/policy.open-cluster-management.io/v1/policygenerator/PolicyGenerator
-                        /policy-generator/PolicyGenerator
+                      - cp /policy-generator/PolicyGenerator-not-fips-compliant /policy-generator-tmp/PolicyGenerator
                       command:
                       - /bin/bash
-                      image: 'registry.redhat.io/rhacm2/multicluster-operators-subscription-rhel8:v{{ (lookup "operator.open-cluster-management.io/v1" "MultiClusterHub" "open-cluster-management" "multiclusterhub").status.currentVersion }}'
+                      image: '{{ (index (lookup "apps/v1" "Deployment" "open-cluster-management" "multicluster-operators-hub-subscription").spec.template.spec.containers 0).image }}'
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-openshift-gitops-policygenerator.yaml:42]: Is there a way to make it work if ACM isn't deployed to open-cluster-management namespace?

File: community/CM-Configuration-Management/policy-openshift-gitops-policygenerator.yaml [hunk 2]
```diff
@@ -36,15 +36,14 @@ spec:
                     initContainers:
                     - args:
                       - -c
-                      - cp /etc/kustomize/plugin/policy.open-cluster-management.io/v1/policygenerator/PolicyGenerator
-                        /policy-generator/PolicyGenerator
+                      - cp /policy-generator/PolicyGenerator-not-fips-compliant /policy-generator-tmp/PolicyGenerator
```
Comment (gparvin) [inline community/CM-Configuration-Management/policy-openshift-gitops-policygenerator.yaml:39]: Does the not fips compliant image exist yet?

File: community/CM-Configuration-Management/policy-openshift-gitops-policygenerator.yaml (hunk 1 above)
Comment (dhaiducek) [reply community/CM-Configuration-Management/policy-openshift-gitops-policygenerator.yaml:42]: Hmmm. Interesting idea--I suppose we could do a `lookup` using a label on the Deployment. That should only return one result that we can parse...

File: community/CM-Configuration-Management/policy-openshift-gitops-policygenerator.yaml (hunk 2 above)
Comment (dhaiducek) [reply community/CM-Configuration-Management/policy-openshift-gitops-policygenerator.yaml:39]: Ah, good point. Though we can merge this now and wait to cherry-pick it to 2.10.

**Review by yiraeChristineKim — APPROVED** [review]

**Pull request merged**